gulpfile.js

```diff
@@ -186,12 +186,9 @@ gulp.task(
 
 gulp.task("lint", function (done) {
   var runExternalModule = require("terriajs/buildprocess/runExternalModule");
-
-  runExternalModule("eslint/bin/eslint.js", [
-    "-c",
-    path.join(getPackageRoot("terriajs"), ".eslintrc.js"),
-    "--ignore-pattern",
-    "lib/ThirdParty",
+  const eslintDir = path.dirname(require.resolve("eslint/package.json"));
+  const eslintExecutable = path.join(eslintDir, "bin", "eslint.js");
+  runExternalModule(eslintExecutable, [
```

Comment:
Locating ESLint is done in the same way as the terriajs repository, and the ignore-pattern has been moved into the config file.

I switched to using the config file in this repository, which just extends the terriajs config file. I'm guessing this "standard" setup makes ESLint more likely work out of the box in IDEs.